WhenPushComesToShove/Assets/3-Scripts/Player Scripts/PlayerLightShoveScript.cs

```diff
@@ -30,4 +35,14 @@ public IEnumerator HitboxCooldown()
         yield return new WaitForSeconds(cooldown);
         hitbox.SetActive(false);
     }
+
+    public void EnableBaseLightShove()
+    {
+        onLightShove += EnableShoveBaseHitbox;
+    }
+
+    public void DisableBaseLightShove()
+    {
+        onLightShove = null;
```

Comment:
Correct me if I'm wrong, but does EnableShoveBaseHitbox need to be unscribed here?

Reply:
By setting it to null, I unsubscribe all attached events including EnableShoveBaseHitbox.